test/e2e/lexicon-new-project.spec.ts

```diff
@@ -0,0 +1,537 @@
+
+    test.beforeEach(async () => {
+      await newLexProjectPageMember.goto();
+    });
+
+    test('Cannot see Back or Next buttons', async () => {
+      await expect(newLexProjectPageMember.backButton).not.toBeVisible();
+      await expect(newLexProjectPageMember.nextButton).not.toBeVisible();
+      await newLexProjectPageMember.expectFormStatusHasNoError();
+    });
+
+    test('Can navigate to new project form and back', async () => {
+      await expect(newLexProjectPageMember.chooserPage.createButton).toBeEnabled();
+      await newLexProjectPageMember.chooserPage.createButton.click();
+      await expect(newLexProjectPageMember.namePage.projectNameInput).toBeVisible();
+
+      // Can go back to Chooser page
+      await expect(newLexProjectPageMember.backButton).toBeVisible();
+      await newLexProjectPageMember.backButton.click();
+      await expect(newLexProjectPageMember.chooserPage.sendReceiveButton).toBeVisible();
+    });
+
+    test('Can navigate to Send and Receive form and back', async ({ member }) => {
+      await expect(newLexProjectPageMember.chooserPage.sendReceiveButton).toBeEnabled();
+      await newLexProjectPageMember.chooserPage.sendReceiveButton.click();
+      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toBeVisible();
+      expect(await newLexProjectPageMember.srCredentialsPage.loginInput.inputValue()).toEqual(member.username);
+      await expect(newLexProjectPageMember.srCredentialsPage.passwordInput).toBeVisible();
+      await expect(newLexProjectPageMember.srCredentialsPage.projectSelect).not.toBeVisible();
+
+      // Can go back to Chooser page
+      await expect(newLexProjectPageMember.backButton).toBeVisible();
+      await newLexProjectPageMember.backButton.click();
+      await expect(newLexProjectPageMember.chooserPage.sendReceiveButton).toBeVisible();
+    });
+  });
+
+  // step 1: send receive credentials
+  test.describe('Send Receive Credentials page', () => {
+
+    test.beforeEach(async () => {
+      await newLexProjectPageMember.goto();
+      await newLexProjectPageMember.chooserPage.sendReceiveButton.click();
+    });
+
+    test('Cannot move on if Password is empty', async () => {
+      await newLexProjectPageMember.expectFormStatusHasNoError();
+      await expect(newLexProjectPageMember.nextButton).toBeEnabled();
+      await newLexProjectPageMember.nextButton.click();
+
+      await expect(newLexProjectPageMember.srCredentialsPage.loginInput).toBeVisible();
+      await expect(newLexProjectPageMember.srCredentialsPage.projectSelect).not.toBeVisible();
+      await newLexProjectPageMember.expectFormStatusHasError();
+      await expect(newLexProjectPageMember.formStatus).toContainText('Password cannot be empty.');
+    });
+
+    test('Cannot move on if username is incorrect and can go back to Chooser page, user and password preserved', async ({ member }) => {
+      // passwordInvalid is, incredibly, an invalid password.
```

Comment:
```suggestion
      // passwordValid is, incredibly, an invalid password.
```

Reply:
I agree, would make sense to change that. Copied blindly from the protractor tests.